From 378925d9936b09ed3a85d6c9b96f49a4ef8f75bf Mon Sep 17 00:00:00 2001
From: odubaj <odubaj@redhat.com>
Date: Thu, 11 Oct 2018 10:11:06 +0200
Subject: [PATCH] fixed bug missidentifying glibc locale files

Information about glibc locale files taken from C library <locale/localeinfo.h>
Link here: https://sourceware.org/git/?p=glibc.git;a=blob;f=locale/localeinfo.h;h=68822a63#l32

---
 magic/Magdir/glibc | 20 ++++++++++++++++++++
 magic/Makefile.am  |  1 +
 2 files changed, 21 insertions(+)
 create mode 100644 magic/Magdir/glibc

diff --git a/magic/Magdir/glibc b/magic/Magdir/glibc
new file mode 100644
index 0000000..76b607f
--- /dev/null
+++ b/magic/Magdir/glibc
@@ -0,0 +1,20 @@
+
+#------------------------------------------------------------------------------
+# $File$
+# glibc locale files
+#
+
+0	belong	0x20070920		glibc locale file LC_CTYPE
+0	belong	0x14110320		glibc locale file LC_NUMERIC
+0	belong	0x17110320		glibc locale file LC_TIME
+0	belong	0x17100520		glibc locale file LC_COLLATE
+0	belong	0x11110320		glibc locale file LC_MONETARY
+0	belong	0x10110320		glibc locale file LC_MESSAGES
+0	belong	0x13110320		glibc locale file LC_ALL
+0	belong	0x12110320		glibc locale file LC_PAPER
+0	belong	0x1d110320		glibc locale file LC_NAME
+0	belong	0x1c110320		glibc locale file LC_ADDRESS
+0	belong	0x1f110320		glibc locale file LC_TELEPHONE
+0	belong	0x1e110320		glibc locale file LC_MEASUREMENT
+0	belong	0x19110320		glibc locale file LC_IDENTIFICATION
+
diff --git a/magic/Makefile.am b/magic/Makefile.am
index 69c9dec..d6657a6 100644
--- a/magic/Makefile.am
+++ b/magic/Makefile.am
@@ -105,6 +105,7 @@ $(MAGIC_FRAGMENT_DIR)/gcc \
 $(MAGIC_FRAGMENT_DIR)/gconv \
 $(MAGIC_FRAGMENT_DIR)/geo \
 $(MAGIC_FRAGMENT_DIR)/geos \
+$(MAGIC_FRAGMENT_DIR)/glibc \
 $(MAGIC_FRAGMENT_DIR)/gimp \
 $(MAGIC_FRAGMENT_DIR)/gnome \
 $(MAGIC_FRAGMENT_DIR)/gnu \
-- 
2.17.1

